int64 no_of_clusters
geometry_msgs/Point[] centroids
int64[] ids
float64[] radii
float64[] cyl_radii
std_msgs/ColorRGBA[] colors
